### PR TITLE
Fix usb_power_mode missing initialization

### PR DESCRIPTION
**Pull request by rbiasini**

### CODE REVIEW

File: board/boards/black.h [hunk 1]
```diff
@@ -183,6 +183,9 @@ void black_init(void) {
 
   // init multiplexer
   can_set_obd(car_harness_status, false);
+
+  // init usb power mdoe
```
Comment (geohot) [inline board/boards/black.h]: typo in comment

File: board/boards/white.h [hunk 2]
```diff
@@ -243,8 +243,9 @@ void white_init(void) {
 
   // on PC, set USB power mode to CLIENT
 #ifndef EON
-  white_set_usb_power_mode(USB_POWER_CLIENT);
+  usb_power_mode = USB_POWER_CLIENT;
```
Comment (geohot) [inline board/boards/white.h]: So I wouldn't do it like this. I'd init to NONE and call white_set_usb_power_mode for CLIENT on PC, and CDP on EON.

white_set_usb_power_mode sets usb_power_mode, it shouldn't be set elsewhere!

Comment (rbiasini) [reply board/boards/white.h]: ok, that forces `set_usb_power_mode` to be called. 
Change done. Panda and cereal need to be bumped at the same time in master after this change.

**Comment by geohot** [comment]
lgtm

**Pull request merged**